src/extractEnglishPages.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by Anshul on 3/6/2017.
+ */
+
+const path = require('path');
+const fs = require('fs');
+var franc = require('franc');
+const WARCStream = require('warc');
+var writeStream = fs.createWriteStream('./data/english.wet', { flags : 'w' });
```

Comment:
Yeah, ok, that's fine with me. The results look promising, but there is no point in merging it into the develop branch right now if it's just a test.